spec/requests/api/v0/highlights_controller_spec.rb

```diff
@@ -4,30 +4,33 @@
 
 RSpec.describe Api::V0::HighlightsController, type: :request do
   let(:user_uuid) { '55783d49-7562-4576-a626-3b877557a21f' }
-  let(:source_parent_id) { 'ccf8e44e-05e5-4272-bd0a-aca50171b50f' }
+  let(:scope_id) { 'ccf8e44e-05e5-4272-bd0a-aca50171b50f' }
   let(:source_id) { SecureRandom.uuid }
+  let(:scope_1_id) { SecureRandom.uuid }
+  let(:uuid1) { SecureRandom.uuid }
 
   before { allow(Rails.application.config).to receive(:consider_all_requests_local) { false } }
 
   describe 'GET /highlights' do
-    let!(:highlight1) { create(:highlight, user_uuid: user_uuid) }
-    let!(:highlight2) { create(:highlight, user_uuid: user_uuid) }
-    let!(:highlight3) { create(:highlight) }
-
-    let(:page) { 1 }
-    let(:per_page) { 10 }
-    let(:source_parent_ids) { highlight1.source_parent_ids }
+    let!(:highlight1) { create(:highlight, user_uuid: user_uuid, source_id: source_id, scope_id: scope_1_id) }
+    let!(:highlight2) { create(:highlight, user_uuid: user_uuid,                       scope_id: scope_1_id) }
+    let!(:highlight3) { create(:highlight, user_uuid: user_uuid,                       scope_id: scope_1_id) }
+    let!(:highlight4) { create(:highlight, user_uuid: user_uuid, source_id: source_id, scope_id: scope_1_id, prev_highlight: highlight1) }
```

Comment:
wonder if these should be renamed to better names to describe the utility?  e.g, highlight_with_prev_neighbor, highlight_with_both_neighbors, highlight_with_no_neighbors

Reply:
I don't know if I could handle those line lengths ;-) I'm ok as is but if you want to change in a future PR that'd be ok by me.